test/docfx.Tests/BuildCommandTest.cs

```diff
@@ -169,11 +169,11 @@ public void TestGetConfigWithNoInputAndDocfxJsonExists()
                 new string[]
                 {
                     "",
-                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"7\">Hello Test1</h1>",
-                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"8\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
+                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"7\" sourceendlinenumber=\"7\">Hello Test1</h1>",
+                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"8\" sourceendlinenumber=\"10\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
```

Comment:
I think it was converted to lowercase in the procedure 'overwrite', in the function ParseCore , it will call a function 'LoadHtml(markupResult.Html)  ,and since then the result will be lowercase,but a can't find the method `LoadHtml` in the class HtmlDocument ( [https://msdn.microsoft.com/en-us/library/system.windows.forms.htmldocument(v=vs.110).aspx](https://msdn.microsoft.com/en-us/library/system.windows.forms.htmldocument(v=vs.110).aspx) )

---
In reply to: [76949337](https://github.com/dotnet/docfx/pull/654#discussion_r76949337) [](ancestors = 76949337)